### ISSUE TITLE
ExportImagePixels is broken

### ISSUE THREAD

**Issue by redcap97** [gographics/imagick#85]
I wrote a code like that:

```
package main

import (
  "gopkg.in/gographics/imagick.v2/imagick"
)

func main() {
  var err error

  imagick.Initialize()
  defer imagick.Terminate()

  mw := imagick.NewMagickWand()
  defer mw.Destroy()

  err = mw.SetSize(100, 100)
  if err != nil {
    panic(err)
  }

  err = mw.ReadImage("xc:white")
  if err != nil {
    panic(err)
  }

  _, err = mw.ExportImagePixels(0, 1, 100, 1, "RGB", imagick.PIXEL_DOUBLE)
  if err != nil {
    panic(err)
  }
}
```

But an error of `ExportImagePixels` occurred:

```
panic: Args x, y, cols, and rows produces an invalid region <= 0

goroutine 1 [running]:
panic(0x4148c80, 0xc82000a120)
        /usr/local/Cellar/go/1.6/libexec/src/runtime/panic.go:464 +0x3e6
main.main()
        /Users/redcap/package/go/src/github.com/redcap97/cutlayer/main.go:28 +0x1da
```

It looks like calculation of  `maplen` is wrong.
Please look other language implementation:

+ https://github.com/rmagick/rmagick/blob/master/ext/RMagick/rmimage.c#L5909
+ https://github.com/mkoppanen/imagick/blob/da6045b5dea79dad222105768ef7f9501c23ee30/imagick_class.c#L2607

**Comment by justinfx** [comment]
Please give af767903 (85_Fix_ExportImagePixels) a test?

**Comment by redcap97** [comment]
af76790 seems to have fixed it! Thank you!

**Comment by justinfx** [comment]
Fixed in [v2.2.2](https://github.com/gographics/imagick/tree/v2.2.2) / [v1.1.2](https://github.com/gographics/imagick/tree/v1.1.2)
